src/Contexts/notesActions-context.jsx

```diff
@@ -1,16 +1,17 @@
 import { notesReducer } from "../Reducers/notesActions-reducer";
```

Comment:
Since there will be multiple reducers at future stages of the project, I suggest you export all reducers from a single index.js file that you can create inside the reducers folder.